tests/integrations/server/kv_service.rs

```diff
@@ -682,16 +672,7 @@ fn test_mvcc_flashback_block_rw() {
     let (_cluster, client, ctx) = must_new_cluster_and_kv_client();
     fail::cfg("skip_finish_flashback_to_version", "return").unwrap();
     // Flashback
-    let mut flashback_to_version_req = FlashbackToVersionRequest::default();
-    flashback_to_version_req.set_context(ctx.clone());
-    flashback_to_version_req.set_start_ts(1);
-    flashback_to_version_req.set_commit_ts(2);
-    flashback_to_version_req.version = 0;
-    flashback_to_version_req.start_key = b"a".to_vec();
-    flashback_to_version_req.end_key = b"z".to_vec();
-    let flashback_resp = client
-        .kv_flashback_to_version(&flashback_to_version_req)
-        .unwrap();
+    let flashback_resp = must_flashback_to_version(&client, ctx.clone(), 0, 1, 2);
```

Comment:
I think it is better to be unified with the above ts in `test_mvcc_flashback`
```suggestion
    let flashback_resp = must_flashback_to_version(&client, ctx.clone(), 0, ts + 1, ts + 2);
```